python/neuroglancer/ingest/lib.py

```diff
@@ -75,7 +77,7 @@ def cloudpath_to_hierarchy(cloudpath):
   cloudpath = format_cloudpath(cloudpath)
   return cloudpath.split('/')
 
-def gcloudFileIterator(cloudpaths, keep_files=None, use_ls=True, compress=False):
+def gcloudFileIterator(cloudpaths, keep_files=None, use_ls=False, compress=False):
```

Comment:
use_ls was useful when downloading an entire dataset (or large proportions of one) onto a single machine. Now we do our processing in chunks almost exclusively, so we should turn that off.